frontend/components/markup-modal/path-builder.js

```diff
@@ -0,0 +1,183 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled, { css } from 'styled-components'
+import pathToRegexp from 'path-to-regexp'
+import humanizeString from 'humanize-string'
+import { noAutocorrect, StyledInput } from '../common'
+import {
+  BuilderContainer,
+  BuilderLabel,
+  BuilderCaption,
+} from './builder-common'
+
+const PathBuilderColumn = styled.span`
+  height: 58px;
+
+  float: left;
+  display: flex;
+  flex-direction: column;
+
+  margin: 5px 0;
+
+  ${({ horizPadding }) =>
+    horizPadding &&
+    css`
+      padding-left: ${horizPadding};
```

Comment:
Seems like this is only ever set to `8px`. Just hard code it? `padding: 0 8px`

Reply:
In `renderLiteral()` it's not set so it needs to be conditional. I could do the same as you suggested elsewhere which is to change the parameter to a boolean.